utils/src/lib.rs

```diff
@@ -373,7 +373,7 @@ pub fn decode_revert(error: &[u8], maybe_abi: Option<&Abi>) -> Result<String> {
                     let actual_err = &err_data[..4];
                     if let Ok(decoded) = decode_revert(actual_err, maybe_abi) {
                         // it's a known selector
-                        return Ok(decoded)
+                        return Ok(decoded);
```

Comment:
let's not have these changes in if possible
```suggestion
                        return Ok(decoded)
```

Reply:
Removed